azext_iot/central/_help.py

```diff
@@ -352,6 +352,28 @@ def _load_central_devices_help():
             --simulated
     """
 
+    helps[
+        "iot central device update"
+    ] = """
+        type: command
+        short-summary: Update a device in IoT Central.
```

Comment:
Is there a limit to what a user can update? For example, they cannot update the device-id, correct?

Reply:
added long-summary